src/main/java/com/sda/triangles/TriangleCalculationApp.java

```diff
@@ -0,0 +1,19 @@
+package com.sda.triangles;
+
+public class TriangleCalculationApp {
+    public static void main(String[] args) {
+        TriangleCalculation triangle = new TriangleCalculation();
+
+        try {
+            System.out.println(triangle.calculateTriangleCircuit(9, 8, 3));
+
+            System.out.println(triangle.calculateTriangleCircuit(0,11,5));
+
+
+        } catch (CannotBuiltTriangleException exception) {
+            System.out.println(exception.getMessage());
+            triangle.wrongSides();
```

Comment:
Można uniknąć obecności tej metody i sprawić że sam exception zwracał taką wiadomość z metody `getMessage()`. Można to osiągnąć przekazując tę wiadomość przy tworzeniu wyjątku.